components/browser/state/src/main/java/mozilla/components/browser/state/search/RegionState.kt

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.state.search
+
+/**
+ * Data class keeping track of the region of the user.
+ *
+ * @param home The "home" region of the user, which will change slowly if the user stays in the same
```

Comment:
nit: s/slowly/only ...or remove. Reading the docs of current made it clear though.